### PR TITLE
Added timeout to pod webhook Test 1

### PR DESCRIPTION
**Pull request by aliceh**
Added timeout to make sure the cluster is ready for the test, in case selector sync sets needed more time.

### CODE REVIEW

**Comment by whereswaldon** [comment]
@aliceh Can you walk me through the rationale for this change?

File: pkg/e2e/verify/pod_webhook.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ var _ = ginkgo.Describe(podWebhookTestName, func() {
 					"system:authenticated",
 				},
 			})
+			time.Sleep(5 * time.Second)
```
Comment (jeefy) [inline pkg/e2e/verify/pod_webhook.go:48]: Is there a more deterministic way to prevent this failure than a sleep? :thinking:

**Comment by aliceh** [comment]
@whereswaldon I had a suspicion that maybe a selector sync set needs more time for the sync and that's why the webhook was not blocking the dedicated admin user's actions correctly.

File: pkg/e2e/verify/pod_webhook.go (hunk 1 above)
Comment (whereswaldon) [reply pkg/e2e/verify/pod_webhook.go:48]: @aliceh If your hypothesis RE: selector sync sets is true, could we instead directly query whether that sync operation is complete? Does the sync set have a status field that could tell us?

**Pull request closed without merging**